app/src/main/kotlin/com/intive/picover/auth/repository/AuthRepository.kt

```diff
@@ -70,6 +71,15 @@ class AuthRepository @Inject constructor(
 		return userProfile()
 	}
 
+	suspend fun updateUserName(userName: String): Profile {
+		requireUser().updateProfile(
+			userProfileChangeRequest {
+				displayName = userName
+			},
+		).await()
+		return userProfile()
+	}
+
 	private fun requireUser() =
 		firebaseAuth.currentUser!!
```

Comment:
I think we should avoid double-bang operators (!!) as it might cause NullPointerException

Reply:
I created an issue for this #170